src/raw/session.rs

```diff
@@ -393,7 +393,7 @@ impl<FS: Filesystem + Send + Sync + 'static> Session<FS> {
         })
     }
 
-    /// mount the filesystem with root permission.
+    /// mount the filesystem without root permission.
```

Comment:
I think the "root permission" part of the comment is misleading.  On FreeBSD, the `nmount` syscall is used in the same way whether or not the process is privileged.  But the end result depends on that privilege: if the process is privileged then the mountpoint's accessibility depends on the `allow_other` mount option.  Or, if the process is unprivileged, then the `allow_other` mount option will be refused, and only the mounting user can access the mount point.
I think the phrase about "root permission" should be removed.